utils/build/webpack.client.prod.js

```diff
@@ -29,17 +30,23 @@ const {
   withWebpackBundleAnalyzerPlugin,
 } = plugins;
 
-const { PRODUCTION } = CONSTANTS;
+const { PRODUCTION, CLIENT_BUILD_DIR, BUNDLE_ANALYSER_DIR } = CONSTANTS;
 
 const prodSpecificConfig = {
   mode: PRODUCTION,
+  output: {
+    path: CLIENT_BUILD_DIR,
+    publicPath: '',
+    filename: '[name].bundle.js',
+  },
   module: {
     rules: [
       withStylingModuleLoader([
         {
           loader: MiniCssExtractPlugin.loader,
         },
       ]),
+      withTSModuleLoader('tsconfig.client.json'),
```

Comment:
We can get rid of JSModuleLoader and add js files to the TSModuleLoader.  TypeScript can handle the javascript files as well.